src/components/Share/style.scss

```diff
@@ -0,0 +1,30 @@
+.share {
+  position: relative;
+  bottom: 2px;
+  left: 5px;
+}
+
+.facebook {
+  background-image: url(../../../static/images/FB-grn.png);
```

Comment:
I was thinking of using it, however I thought there would be push against an additional dependency. I'll use fontawesome.